src/coreComponents/common/format/table/TableLayout.hpp

```diff
@@ -58,65 +58,145 @@ class TableLayout
     /// Name for a column
     string columnName;
     /// Alignment for a column. By default aligned to the right side
-    Alignment alignment = Alignment::right;
+    Alignment alignment;
```

Comment:
As you removed ` = Alignment::right`, `alignment` is no longer initialized if we call 2/4 constructors.